test/e2e/prometheus_test.go

```diff
@@ -4657,7 +4659,7 @@ func testRelabelConfigCRDValidation(t *testing.T) {
 			relabelConfigs: []*monitoringv1.RelabelConfig{
 				{
 					SourceLabels: []monitoringv1.LabelName{"__address__"},
-					Separator:    ",",
+					Separator:    &sep1,
```

Comment:
(nit) you can use ptr.To instead.

Reply:
Agree, using pointer package from k8s would be better.

Reply:
I'll fix this nit tomorrow when I have more time.